source-http-file/main.go

```diff
@@ -200,6 +200,13 @@ func main() {
       }`)
 		},
 		DocumentationURL: "https://go.estuary.dev/source-http-file",
+		// Set the delta to far in the future. The scenario that the the `MaxBound` protects agains
```

Comment:
I think this should be zero, and not in-the-future.

If this were an hour in the future:
* Sweep zero starts at now0, and runs with a minBound of 0 and an upperbound of now0+1h.
* The file has a last-modified of now0-2s and is processed.
* The sweep completes, and the new minBound is now0+1h.
* Sweep one starts shortly after at now1, very slightly larger than now0.
* It runs with minBound of now0+1h, and maxBound of now1+1h.
* It sees a file with last-modified of now1-3s _which is outside of the bounds_.

tl;dr after the first sweep, I don't see how the file is processed again.

Reply:
We don't update the minBound based on the maxBound, but rather based on the greatest modTime of the files actually processed during the sweep. This is done [here](https://github.com/estuary/connectors/blob/10e168b5bbf956561e0e614f309bd0791c5efff7/filesource/state.go#L111). So I _think_ we're good, but LMK if there's something I'm missing.

Reply:
Indeed! Yea I was wrong, I think this is good too.